src/index.ts

```diff
@@ -109,10 +111,57 @@ export class EppoJSClient extends EppoClient {
     isObfuscated: true,
   });
 
+  /**
+   * @deprecated
```

Comment:
alternative?

Reply:
undeprecated. I want to make a public `isInitialized`, but more work to the underlying config handling is needed first